.gitignore

```diff
@@ -5,4 +5,5 @@
 .vscode
 .pytest_cache
 .ds_store
-bdnyc/
\ No newline at end of file
+bdnyc/
+/test.py
```

Comment:
Tell me more about why you're editing the .gitignore.

Reply:
I was not meant to commit that change, sorry. It's a remote file I was using to test certain aspects of the code, I'll try to revert the commit ASAP.